components/ds3231/ds3231.c

```diff
@@ -87,6 +87,30 @@ static uint8_t dec2bcd(uint8_t val)
     return ((val / 10) << 4) + (val % 10);
 }
 
+#include <stdio.h>
```

Comment:
move this `#include` to the up. All `includes` must be grouped in one place

Reply:
Include moved